data-loader/core/src/test/java/com/scalar/db/dataloader/core/util/RuntimeUtilTest.java

```diff
@@ -0,0 +1,24 @@
+package com.scalar.db.dataloader.core.util;
+
+import static com.scalar.db.common.error.CoreError.DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+
+import org.junit.jupiter.api.Test;
+
+/** RuntimeUtils unit tests */
+public class RuntimeUtilTest {
+
+  @Test
+  public void checkNotNull_HasNullValues_ShouldThrowException() {
+    assertThatThrownBy(() -> RuntimeUtil.checkNotNull(null, null))
+        .isExactlyInstanceOf(NullPointerException.class)
+        .hasMessage(DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT.buildMessage());
+  }
+
+  @Test
+  public void checkNotNull_HasNoNullValues_ShouldNotThrowException() {
+    String string = "1";
+    Object object = new Object();
+    RuntimeUtil.checkNotNull(string, object);
+  }
+}
```

Comment:
For consistency:
```suggestion
class RuntimeUtilTest {

  @Test
  void checkNotNull_HasNullValues_ShouldThrowException() {
    assertThatThrownBy(() -> RuntimeUtil.checkNotNull(null, null))
        .isExactlyInstanceOf(NullPointerException.class)
        .hasMessage(DATA_LOADER_ERROR_METHOD_NULL_ARGUMENT.buildMessage());
  }

  @Test
  void checkNotNull_HasNoNullValues_ShouldNotThrowException() {
    String string = "1";
    Object object = new Object();
    RuntimeUtil.checkNotNull(string, object);
  }
}
```

Reply:
Thank you. 
I have updated it as suggested to make it consistent.